anydex/core/tickentry.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, tick, price_level):
         :type tick: Tick
         :type price_level: PriceLevel
         """
-        super(TickEntry, self).__init__()
+        super().__init__()
```

Comment:
You can remove this line right?

Reply:
I tried and it looks like it's necessary.

This is a test done without the `super()` call: https://jenkins-ci.tribler.org/job/AnyDex/job/test_anydex_PR_linux/126/console
This is a test done with: https://jenkins-ci.tribler.org/job/AnyDex/job/test_anydex_PR_linux/127/console

Reply:
You're right; I forgot that `TickEntry` is a subclass of `TaskManager` so then it's required 👍